static/app/components/badge/groupPriority.tsx

```diff
@@ -201,6 +203,12 @@ export function GroupPriorityDropdown({
           {...triggerProps}
           aria-label={t('Modify issue priority')}
           size="zero"
+          disabled={disabled}
+          title={
+            disabled
+              ? t('Manually updating the priority of this issue is disabled.')
```

Comment:
maybe this wording could be more specific to metric issues?

```suggestion
              ? t('You cannot manually update the priority of a metric issue.')
```